Training.Core/Models/Album.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Training.Core.Contracts;
+
+namespace Training.Core.Models
+{
+    public class Album : IRemovable
+    {
+
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public bool IsDeleted { get; set; }
+
+        public ICollection<Book> Books { get; set; }
```

Comment:
Perhaps we are having a different concept of "Album", I see like a music album, and being something completly isolated, I think the relation with Book is not necessary

However, despite that for me functionally it has no sense, technically speaking, the relation with EFCore has been correctly done ;)